### ISSUE TITLE
Update some tests to use equality rather than identity comparison

### ISSUE THREAD

**Issue by dellgreen** [intel/bmap-tools#68]
New checks/warnings seem to of come in with python3.8 which emit syntax warnings on some tests the first time the python code is compiled. Should be quick and minor issue to fix.

https://bugs.python.org/issue34850

nosetests-3.4 tests/test_filemap.py

```
/home/dell/Programming/projects/upstream/bmap-tools/tests/test_filemap.py:58: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if ranges_type is "mapped":
/home/dell/Programming/projects/upstream/bmap-tools/tests/test_filemap.py:60: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif ranges_type is "unmapped":
/home/dell/Programming/projects/upstream/bmap-tools/tests/test_filemap.py:96: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if ranges_type is "mapped" and filemap.block_is_unmapped(block):
/home/dell/Programming/projects/upstream/bmap-tools/tests/test_filemap.py:100: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if ranges_type is "unmapped" and filemap.block_is_mapped(block):
```

**Comment by dedekind** [comment]
Dell, I guess you could just submit a small fix :-)

**Comment by dellgreen** [comment]
looks like you already fixed it in 50c9946 :)
